### ISSUE TITLE
Editing figure labels

### ISSUE THREAD

**Issue by PhilippMayr** [OSCOSS/fiduswriter#77]
it is very unhandy to edit figure labels. Now you need to select the figure and press the button figure. then editing is possible.

It would be better to directly edit the label by clicking on the label

**Comment by johanneswilm** [comment]
@PhilippMayr This will be partially addressed by Niloofar's work on https://github.com/OSCOSS/fiduswriter/issues/72 which should make it easier to understand what is going on and how to change the label and other figure information.

However, a change where the entire figure could configured "in place" (including the label text) will require more fundamental changes to the document and possibly browsers. The ProseMirror discussion group has been discussing this for a while and they have done some experiments, but I am not sure how quickly this will get stable enough for everyday usage. The main problem with allowing the editing of the label in place is that this creates an editable "lake" on a non-editable "island", and the browsers still don't know how to behave well when the user selects from the lake to the sea adn then hits "delete".

In short: This may possibly have to be clasified as a longterm goal.
